Sources/Public/Animation/LottieAnimationHelpers.swift

```diff
@@ -164,6 +164,29 @@ extension LottieAnimation {
     }
   }
 
+  /// Loads a Lottie animation asynchronously from the URL.
+  ///
+  /// - Parameter url: The url to load the animation from.
+  /// - Parameter animationCache: A cache for holding loaded animations. Defaults to `LottieAnimationCache.shared`. Optional.
+  ///
+  @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
+  public static func loadedFrom(
```

Comment:
I added `async` methods for loading `DotLottieFile`s so figured we may as well also add this one for remote `LottieAnimation`s

Reply:
Nice!